Python/Product/Common/Strings.resx

```diff
@@ -2909,4 +2909,15 @@ Please specify at least one package.</value>
   <data name="WorkspaceText" xml:space="preserve">
     <value>workspace</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="PythonTestDiscovererStartedMessage" xml:space="preserve">
+    <value>Starting {0} discovery for the {1} {2}. This can take up to {3} seconds.</value>
+    <comment>{0} is the testframe name "pytest" or "unittest"
+{1} is the project or workspace name
+{2} is "project" or "workspace"
+{3} is an integer wait time in seconds</comment>
+  </data>
+    <data name="PythonTestDiscovererTimeoutErrorMessage" xml:space="preserve">
+    <value>Python test discovery timeout, possibly due to blocking call. Go to https://aka.ms/VSTestExplorerPython for more details on test discovery.</value>
+  </data>
+</root>
+
```

Comment:
new strings example
Note. this was using 1 seconds, but normally 60
![image](https://user-images.githubusercontent.com/1946977/62240260-6a7e5100-b38b-11e9-809d-40cd07de1e6c.png)